common/test/acceptance/tests/discussion/test_discussion.py

```diff
@@ -425,6 +432,26 @@ def test_discussion_blackout_period(self):
         self.assertFalse(self.thread_page.is_comment_deletable("comment1"))
         self.assertFalse(self.thread_page.is_comment_deletable("comment2"))
 
+    def test_dual_discussion_module(self):
+        """
+        Scenario: Two discussion module in one unit shouldn't override their actions
+        Given that I'm on courseware page where there are two inline discussion
+        When I click on one discussion module new post button
+        Then I should be shown new post form of that module
+        and it shouldn't open other discussion module present there.
+        """
+        self.discussion_page.wait_for_page()
+        self.additional_discussion_page.wait_for_page()
+        self.discussion_page.click_new_post_button()
```

Comment:
OK, please add some comments in the test code itself to explain why you are doing all of these steps (which seem unnecessary given the abbreviated BDD description, but are actually necessary to catch the bug).